source/lib/api/AST/ParseJavadoc.cpp

```diff
@@ -334,6 +334,21 @@ class JavadocVisitor
             param.name = ensureUTF8(C->getParamNameAsWritten().str());
         else
             param.name = "@anon";
```

Comment:
"@anon" ?

Reply:
It's used for `@param` directives without any parameter name provided... only place I could see it being useful is for variadic function a la `void f(...)`.